src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,43 @@
+package coffee.order;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class CoffeeOrderBoard {
+    private List<Order> orders = new ArrayList<>();
```

Comment:
It's not a good idea to use ArrayList as a queue as on remove first element whole list is recalculated. Consider using the any implementation of Queue interface